### PR TITLE
TC 3.0.2

### PR DESCRIPTION
**Pull request by f-peverali**
# Version Upgrade Template

Version: 3.0.2

Date: 30.1.2024

## Description

This is a Pullreuqest that requires an increase in the Version number. Therefore, multiple outside-github, related Task have to be performed and checked.

All jobs with an `x` in the boxes were performed to the best of knowledge.

## Pre-Merge Activities

- [x] This PR refers to a versioned Branch with a name and a version number in the form of N.n.n, e.g. "TC_3.2.1".
- [x] This PR has a clean meaningful commit history. Minor commits or commits without description have been squashed, at the latest now.
- [x] The _./github/workflows/main.yml_ refers to the correct Firely Terminal and SUSHI Version. 
    > **_Firely Terminal Pipeline_**  0.4.0.

    > **_SUSHI Versions_**  3.5.0.
- [x] By running the _Release_Publish.py_ script, release version and date was updated accordingly. The script ran without errors.
- [x] Eventually, increase the dependency of to newer Basis Modul (package and sushi-config)
- [x] New Release Notes were created, alined to the commit history and cleaned. In Github, go to 
  - [x]  _-> Releases_ then _-> Draft a new release_ with the _Modul Name and Version_, then
  - [x] _-> Target the main-Branch_ and _->enter a new Tag according to the Version_, then click.
  - [x] Click _-> Generate Release notes_ , _->Adjust them if necessary_ and _-> Copy/Paste the Details in the RealeaseNotes.md_ of the very Branch you want to merge.
  - [x] Finally _-> Save as Draft_

## Merge and Publishing

- [x] With the updated Version, Dates, and Release Notes (as described above) with the last committ into the Branch you want to merge.
- [x] In GitHub _-> Actions_ the _->CI (FHIR Validation)_ workflow terminates successfully.
- [x] Add the Approve / the PR gets positively reviewed by a colleague.
- [x] Merge (without squash) the PR, delete the Branch.

## Post-Merge Activities

- [x] Go to the corresponding SIMPLIFIER Project and _-> Github -> Reimport_ the project.
- [x] Go to the corresponding SIMPLIFIER Project and _-> Packages -> Expand the Dropdown for Create -> Create new package_ for the project.
  - [x] With the corresponding version number, and
  - [x] The Release notes (from above) and a compare-link to the previous Release.
  - [x] Unlist the old package by _-> clicking on the old package_, _-> go to Administration_ and _-> click on Unlist_
- [x] Publish the previously drafted Release, including version number, on GitHub.
- [ ] Provide / Archive the IG in the corresponding _gh-pages_ branch of the GitHub project.
  - [x] Checkout the Branch (no need to merge it later).
  - [x] Export from Simplifier via _-> Guides -> Expand the Modul ... -> Export_
  - [x] Unpack the zip, remove the packages folder (because its kinda big), and move everything else to a (version corresponding) new folder in the branch folder structure.
  - [x] Update the file index.html and check rendering
  - [x] commit the branch.
- [x] If ISiK Basismodul was updated all depending Modules should be updated  with a renewed dependency to the incremented Basismodul version - possibly including and closing technical corrections

### CODE REVIEW

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
 
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+
+Version: 3.0.2
+Datum: 30.01.2023
+
+* MS auf dataAbsentReason gesetzt und Hinweis zur Nutzung https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/144
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md:12]:
```suggestion
* Informationen zu Profilen präzisiert und Verpflichtung bei GCS und EKG geschwächt: https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/145/commits/3643ece93148c721d88c31ee1f6dbf8b5cbdd683
* MS auf dataAbsentReason gesetzt und Hinweis zur Nutzung: https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/144 
```

File: ImplementationGuide/markdown/Ekg.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # EKG
-Mit diesem Profil lässt sich ein Elektrokardiogramm erfassen.
+Mit diesem Profil lässt sich ein Elektrokardiogramm erfassen und kann dazu genutzt werden kurze EKGs abzubilden. Es ist nicht dafür gedacht Langzeitmonitoring-EKGs zu repräsentieren.
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Ekg.md:2]: suggestion (blocking): ich würde es eher so formulieren: "Systeme, welche EKG-Daten mittels FHIR verarbeiten, SOLLEN dieses Profil nutzen."

@patrick-werner

File: ImplementationGuide/markdown/Ekg.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
-{{tree:https://gematik.de/fhir/isik/v3/VitalparameterUndKoerpermasze/StructureDefinition/ISiKEkg, hybrid}}
-
-## Beispiele
-
-Valides Minimalbeispiel für das Profil ISiKEkg:
-
+# EKG
+Mit diesem Profil lässt sich ein Elektrokardiogramm erfassen und kann dazu genutzt werden kurze EKGs abzubilden. Es ist nicht dafür gedacht Langzeitmonitoring-EKGs zu repräsentieren.
+
+Alle bestätigungsrelevanten Systeme für dieses Modul, welche EKGs verwalten, SOLLEN Ressourcen verarbeiten können, die zu diesem Profil konform sind.
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Ekg.md]:
```suggestion
Bestätigungsrelevante Systeme, welche EKG-Daten mittels FHIR verarbeiten, SOLLEN dieses Profil nutzen.

```

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 4]
```diff
@@ -4,6 +4,15 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
...
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+
+Version: 3.0.2
+Datum: 30.01.2023
+
+
+* MS auf dataAbsentReason gesetzt und Hinweis zur Nutzung https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/144 
+* ISiK-EKG: .encounter MS Flag hinzu : https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/146
+* Informationen zu Profilen präzisiert und Verpflichtung bei GCS und EKG geschwächt: https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/145/commits/3643ece93148c721d88c31ee1f6dbf8b5cbdd683
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
* Informationen zu Profilen präzisiert und Verpflichtung bei GCS und EKG geschwächt: https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/145
```

File: Resources/sushi-config.yaml
Comment (f-peverali) [inline Resources/sushi-config.yaml:1]: issue: 3.0.4 ist status quo , ggf. auch in der pipeline 3.0.5

File: package.json
Comment (f-peverali) [inline package.json:1]: issue: BAsis 3.0.4 ist status quo , ggf. auch in der pipeline 3.0.5

**Review by patrick-werner — APPROVED** [review]

**Pull request merged**